gmso/formats/prm_writer.py

```diff
@@ -0,0 +1,124 @@
+"""CHARMM Par format."""
+
+from gmso.formats.formats_registry import saves_as
+from gmso.utils.units import LAMMPS_UnitSystems
+
+
+@saves_as(".prm", ".par")
+def write_prm(topology, filename):
+    """Write CHARMM Parameter .prm file given a parametrized structure.
+
+    Notes
+    -----
+    Follows format according to
+    https://www.ks.uiuc.edu/Training/Tutorials/namd/namd-tutorial-unix-html/
+    node25.html
+    Furthermore, ParmEd can support writing CHARMM par, rtf, str files
+    by converting the parmed.Structure into parmed.CharmmParameterSet
+
+    Parmed stores rmin/2 in "rmin"
+    """
+    unit_system = LAMMPS_UnitSystems("real")  # ang, kcal/mol, amu
```

Comment:
Do we need to make sure we're using this everywhere we write out parameters? I see we use it for the atom mass, but not anywhere else.

Reply:
Yep, for in order to handle any input unyts this will do the filtering. I was just too lazy to do it every time at first. Was wondering if you think we should make a system that isn't specific to LAMMPS for this one. It is the exact same as the real units, but maybe a touch confusing to copy them across to other engines. What do you think?

Reply:
I wonder if `unit_system` should be a parameter of `write_prm` that defaults to `None`. If nothing is passed, then all units are used as-is from the toplogy? If we set `unit_system = topology.unit_system` as default behavior, would that work as well?

```
def write_prm(topology, filename, unit_system=None):
    if not unit_system:
        unit_system = topology.unit_system
```

Reply:
The PRM file looks to me like it has default assumed units of kCal/mol, angstrom, amu, radians, etc for it's units. So it can only convert to one defined system.